### ISSUE TITLE
Profile can't be added or used

### ISSUE THREAD

**Issue by sarah12345** [99designs/aws-vault#380]
I have my credentials and profiles setup in ~/.aws/credentials and ~/.aws/config files. When I run `aws-vault add xxx`, I get a response of `Your profile has a source_profile of read-only, adding credentials to xxx won't have any effect`.

When I try to run `aws-vault login xxx`, I get `Failed to get credentials for read-only (source profile for xxx): The specified item could not be found in the keyring.`

I have tried setting different values for AWS_VAULT_BACKEND and running the commands again and still get the same result. 

However, if I add a profile that is not already in my credentials file, it will let me add via the add command, and then login with it.

**Comment by artburkart** [comment]
I ran into a problem with the same symptoms and @sarah12345's solution helped me out. 

I have the following env vars set:
```shell
export AWS_VAULT_KEYCHAIN_NAME=login
export AWS_VAULT_PROMPT=osascript
```
I think I caused the trouble on my laptop because I had 8 instances of `aws-vault` running in parallel, and I killed them all with a `SIGNINT`. Could this have led to all my credential entries being lost?

To fix it, I manually added my values to the keyring like such:
<img width="425" alt="grafik" src="https://user-images.githubusercontent.com/463528/66866001-b60ec700-ef98-11e9-9688-bb4a3de80efb.png">
And then I deleted them (it didn't matter whether I did it through Keychain Access or through `aws-vault`). 

Afterward, everything was back to normal.
